main.go

```diff
@@ -1318,7 +1318,7 @@ func main() {
 			return
 		}
 		for _, entry := range entries {
-			if !entry.Mode().IsRegular() || !strings.HasSuffix(entry.Name(), ".json") {
+			if !entry.Mode().IsRegular() || !strings.HasSuffix(entry.Name(), ".yaml") {
 				// Only inspect JSON files.
```

Comment:
that commands also wants an update :)

Reply:
No, that’s for `go list` if I understood correctly

Reply:
Oh, wait, probably you are right, thanks!

Reply:
Fixed